### ISSUE TITLE
Swagger ui bundle can't be used twice with the same result

### ISSUE THREAD

**Issue by Mortega5** [swagger-api/swagger-ui#4772]
### Q&A
 - OS: linux
 - Browser: chrome 
 - Version: 67.0.3396.99
 - Method of installation: npm
 - Swagger-UI version: swagger-ui-dist 3.17.6
 - Swagger/OpenAPI version: OpenApi 3.0

### Content & configuration

Example Swagger/OpenAPI definition:
* Yaml file is available [here](https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore-expanded.yaml)

Swagger-UI configuration options:
```js
    SwaggerUIBundle({ 
      url: 'https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore-expanded.yaml',
      domNode: $('#test')
    })
```

### Describe the bug you're encountering
The first time you instance the swagger ui, it works correctly. After this, if you do it using SwaggerUIBundle variable again, it doesn't show the same information. For example the requestBody of each request and OAS3 badge are missing. This always happens until you reload the page. It is a big problem for SPA pages.

### To reproduce...

1. Go to https://jsfiddle.net/mortega5/5wqcmvoL/3/'
2. Click on 'load swagger'
3. Click on it again

### Expected behavior
It should display the same information regardless the number of calls. It should not store any information about the first time it was invoked.

### Screenshots
*First time you load it*
![image](https://user-images.githubusercontent.com/8302870/43468436-46da206a-94e4-11e8-81e3-e91e0d44f444.png)

*Second time*
![image](https://user-images.githubusercontent.com/8302870/43468453-5612679a-94e4-11e8-98d4-8b5702e1a013.png)

**Comment by Mortega5** [comment]
I checked that it only happens using openapi 3.x. If I use version 2.0, it works properly.

[Try it](https://jsfiddle.net/mortega5/ytvzq6ro/)

First time you load it
![image](https://user-images.githubusercontent.com/8302870/44203530-86272200-a14f-11e8-9ae1-4ee41c5de5ff.png)

Second time
![image](https://user-images.githubusercontent.com/8302870/44203543-90e1b700-a14f-11e8-9487-7b28926c2149.png)

**Comment by bzd2000** [comment]
I have the same issue, I somebody looking at this?

**Comment by shockey** [comment]
This should be fixed (see #4923) in the latest version, which was released yesterday.

Thanks everyone!
